client/src/index.js

```diff
@@ -1,6 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware } from 'redux';
+import reducers from './reducers';
+
 import App from './App';
 
+const store = createStore(reducers, {}, applyMiddleware());
```

Comment:
you guys are not using any middleware yet is there a different reason for having `applyMiddleware()` ?

Reply:
We're just setting up redux..middleware will come in next PR..please bare with us 😸

Reply:
Ohh jesus, adding unused code to production.....
but let's just say for scalibility and it'll be fine

Reply:
@Karyum Thanks for your patience Mario! we are just learning how to do this but promise we will get better :) I think we pretty much did all the changes you requested besides the ones we commented on... hope u will be happy now ;)

Reply:
@MynahMarie lol uno problemo there is no commit yet for the changes :rofl:

Reply:
@Karyum Oh man sorry yeah we just fixed locally and didnt realize that @MarlenAw hadn't pushed the changes yet 😄 I think I need another coffee.. anyway, changes pushed now!